File: src/testingil/webinar/tddlegacycode/e3/addfunctionality/s1/refactor/s0/CalculatorStartTests.java
package testingil.webinar.tddlegacycode.e3.addfunctionality.s1.refactor.s0;

import static org.junit.jupiter.api.Assertions.*;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

class CalculatorStartTests extends BaseCalculatorTests{

	@Test
	void when_pressing_plus_should_show_0() {
		pressing("+").shouldShow("0");
	}

	@Test
	void when_pressing_div_should_show_0() {
		pressing("/").shouldShow("0");
	}

	@Test
	void when_pressing_equal_should_show_0() {
		pressing("=").shouldShow("0");
	}
}

File: C#/E3.AddFunctionality/S1.Refactor/S1.guard/CalculatorStartTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace TDDLegacyCode.E3.AddFunctionality.S1.Refactor.S1.Guard
{
    [TestClass]

    public class CalculatorStartTests : BaseCalculatorTests
    {
        [TestMethod]
        public void When_pressing_operation_should_show_0()
        {
            Pressing("+").ShouldShow("0");
        }

        [TestMethod]
        public void When_pressing_div_should_show_0()
        {
            Pressing("/").ShouldShow("0");
        }

        [TestMethod]
        public void When_pressing_equal_should_show_0()
        {
            Pressing("=").ShouldShow("0");
        }
    }
}
